src/dndbeyond.pyj

```diff
@@ -613,7 +613,9 @@ class Monster(CharacterBase):
         for ability in self._abilities:
             if ability[1] == "DEX":
                 modifier = ability[3]
-                sendRoll(self, "initiative", "1d20" + modifier, {"initiative": modifier})
+                score = ability[2]
+                roll_modifier = float(modifier) + float(score / 100) if self.getGlobalSetting("initiative-with-dex-score", False) else modifier

```

Comment:
Due to the length of the line, please split it into :
```python
if self.getGlobalSetting("initiative-with-dex-score", False):
  score = ability[2]
  roll_modifier = float(modifier) + float(score / 100)
else:
  roll_modifier = modifier
```
to make it easier to read. I usually reserve the one-liner `x if y else z` constructs for simpler lines only.

Reply:
Seeing the later cast of the modifier into a string from the roll20 side, this reminds me that the roll modifier here should be a string, and not a float, so make sure you cast it into a string in here.

Reply:
Do you prefer the stylistic:
```
initiative = modifier
if self.getGlobalSetting("initiative-with-dex-score", False):
  score = ability[2]
  initiative = str(float(modifier) + float(score / 100))
```

Reply:
yes, actually, I do prefer it the way you just described. Leaving optional settings to just change the behavior rather than define it. Especially since the default would be `initiative = modifier`
Good catch. Thanks!